t/t1000-print-hierarchy.t

```diff
@@ -1,87 +1,89 @@
 #!/bin/bash
 
 test_description='Test print-hierarchy command'
+
 . `dirname $0`/sharness.sh
-PRINT_HIERARCHY=${FLUX_BUILD_DIR}/src/fairness/print_hierarchy/flux-account-shares
 SMALL_NO_TIE=${SHARNESS_TEST_SRCDIR}/expected/test_dbs/small_no_tie.db
 SMALL_TIE=${SHARNESS_TEST_SRCDIR}/expected/test_dbs/small_tie.db
 SMALL_TIE_ALL=${SHARNESS_TEST_SRCDIR}/expected/test_dbs/small_tie_all.db
 OUT_OF_INSERT_ORDER=${SHARNESS_TEST_SRCDIR}/expected/test_dbs/out_of_insert_order.db
 
+flux setattr log-stderr-level 1
+
```

Comment:
This will likely fail when not after `test_under_flux` (or, it will attempt to setattr in the enclosing instance if there is one, e.g. the system instance. Probably not what you want)

Reply:
Ah, TIL. Thanks for explaining what that line does - I just squashed and pushed a fix to remove the `flux setattr log-stderr-level 1` line from `t1000-print-hierarchy.t`.